src/components/dialogs/network-modifications/shunt-compensator/shunt-compensator-dialog.type.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import {
+    ADDITIONAL_PROPERTIES,
+    BUS_OR_BUSBAR_SECTION,
+    CHARACTERISTICS_CHOICE,
+    CONNECTED,
+    CONNECTION_DIRECTION,
+    CONNECTION_NAME,
+    CONNECTION_POSITION,
+    CONNECTIVITY,
+    EQUIPMENT_ID,
+    EQUIPMENT_NAME,
+    ID,
+    MAX_Q_AT_NOMINAL_V,
+    MAX_SUSCEPTANCE,
+    MAXIMUM_SECTION_COUNT,
+    SECTION_COUNT,
+    SHUNT_COMPENSATOR_TYPE,
+    SWITCHED_ON_Q_AT_NOMINAL_V,
+    SWITCHED_ON_SUSCEPTANCE,
+    VOLTAGE_LEVEL,
+} from '../../../utils/field-constants';
+import { Property } from '../common/properties/property-utils';
+import { ConnectablePositionFormInfos } from '../../connectivity/connectivity.type';
+
+export type ShuntCompensatorDialogSchemaBaseForm = {
```

Comment:
```suggestion
export type ShuntCompensatorSchemaBaseForm = {
```
I don't think we need to have "Dialog" in the type name. A bit like `LoadModificationSchemaForm` in `load-modification.type.ts`

Same for the Form in `ShuntCompensatorFormInfos`. It is not included in `LoadModificationInfos` in `load-modification.type.ts`